templates/paragraph/paragraph--localgov-call-out-box.html.twig

```diff
@@ -1,16 +1,27 @@
+{%
+  set classes = [
+    'call-out-box',
+    colour_theme ? 'call-out-box--' ~ colour_theme|clean_class,
+    'row',
+    'mx-1'
+  ]
+%}
 
-<div class="call-out-box row mx-1">
-  <div class="col-12 col-md-6 order-md-2 p-0" />
-    {% if background_url is defined and background_url %}
-      <img src="{{ background_url }}" />
-    {% endif %}
-  </div>
+<div{{ attributes.addClass(classes) }}>
+  {% if background_url is defined and background_url %}
+    <div class="col-12 col-md-6 order-md-2 p-0">
+      <img src="{{ background_url }}" alt="{{ image_alt_text }}" />
+    </div>
+  {% endif %}
   <div class="col-12 col-md-6 order-md-1 p-4">
-    <h2>{{ content.localgov_header_text }}</h2>
+    {{ heading }}
```

Comment:
can you wrap this in an `<h2>` tag like so: `<h2>{{ heading }}</h2>`

Reply:
`heading` element already includes `<hX>` tag (see [https://github.com/localgovdrupal/localgov_campaigns/blob/master/modules/localgov_campaigns_paragraphs/localgov_campaigns_paragraphs.module](https://github.com/localgovdrupal/localgov_campaigns/blob/master/modules/localgov_campaigns_paragraphs/localgov_campaigns_paragraphs.module)) 

It doesn't have to be `<h2>`, it should be a heading level chosen by the content editor (though it is `<h2>` by default)